### PR TITLE
Add Ubuntu 18.04

### PR DESCRIPTION
**Pull request by afgane**

### CODE REVIEW

File: Dockerfile.ubuntu.trusty [hunk 1]
```diff
@@ -12,5 +12,4 @@ ADD local.yml /tmp/ansible/local.yml
 ADD defaults /tmp/ansible/defaults
 ADD tasks /tmp/ansible/tasks
 ADD files/galaxyproject-nginx-pin-600 /tmp/ansible/galaxyproject-nginx-pin-600
-ADD files/galaxyproject-nginx-pin-600 /tmp/ansible/galaxyproject-nginx-pin-600
-RUN ansible-playbook -i localhost, local.yml -e "@defaults/main.yml"
+RUN ansible-playbook -i localhost, local.yml -e "@defaults/main.yml" -e"@defaults/trusty_debian.yml"
```
Comment (drosofff) [inline Dockerfile.ubuntu.trusty:15]: Feel free to fix it also for Dockerfile.ubunty.xenial ;-)

File: defaults/main.yml [hunk 2]
```diff
@@ -30,5 +30,5 @@ postgres_user_home: /var/lib/postgresql
 configure_docker: yes
 install_apparmor: yes
 install_rabbitmq: no
-docker_package: docker-engine
+docker_package: docker-ce
```
Comment (drosofff) [inline defaults/main.yml:33]: just for info as it becomes the default, what is the difference between docker-ce and docker-engine ?

**Review by drosofff — APPROVED** [review]
looks good, thanks !

File: Dockerfile.ubuntu.trusty (hunk 1 above)
Comment (afgane) [reply Dockerfile.ubuntu.trusty:15]: Meant to and then forgot; thanks for the reminder.

File: defaults/main.yml (hunk 2 above)
Comment (afgane) [reply defaults/main.yml:33]: My understanding is that `docker-engine` is the _old_ name that's been replaced with the `docker-ce` variant for the community edition: https://stackoverflow.com/a/45023650/523109
In turn, the `-engine` is not available on more recent operating system versions and vice versa.

Comment (drosofff) [reply defaults/main.yml:33]: Good to know ! thanks

**Pull request merged**